lib/tasks/manageiq/webpacker.rake

```diff
@@ -1,33 +1,34 @@
-if File.exist? 'manageiq-ui-classic.gemspec'
```

Comment:
(And if I'm getting this right, there's no chance of using the original behaviour with the ui as a gem.)